frame/election-provider-multi-phase/src/weights.rs

```diff
@@ -212,86 +185,61 @@ impl WeightInfo for () {
 	// Storage: Staking ForceEra (r:1 w:0)
 	// Storage: ElectionProviderMultiPhase CurrentPhase (r:1 w:0)
 	fn on_initialize_nothing() -> Weight {
-		(22_589_000 as Weight)
+		(23_581_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(8 as Weight))
 	}
-	// Storage: Staking CounterForValidators (r:1 w:0)
-	// Storage: Staking Validators (r:2 w:0)
-	// Storage: Staking CounterForNominators (r:1 w:0)
-	// Storage: Staking SlashingSpans (r:1 w:0)
-	// Storage: Staking Bonded (r:1 w:0)
-	// Storage: Staking Ledger (r:1 w:0)
-	// Storage: Staking Nominators (r:1 w:0)
-	// Storage: Staking ValidatorCount (r:1 w:0)
 	// Storage: ElectionProviderMultiPhase Round (r:1 w:0)
-	// Storage: ElectionProviderMultiPhase SnapshotMetadata (r:0 w:1)
-	// Storage: ElectionProviderMultiPhase DesiredTargets (r:0 w:1)
-	// Storage: ElectionProviderMultiPhase Snapshot (r:0 w:1)
 	// Storage: ElectionProviderMultiPhase CurrentPhase (r:0 w:1)
 	fn on_initialize_open_signed() -> Weight {
-		(107_551_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(10 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
-	}
-	// Storage: Staking CounterForValidators (r:1 w:0)
-	// Storage: Staking Validators (r:2 w:0)
-	// Storage: Staking CounterForNominators (r:1 w:0)
-	// Storage: Staking SlashingSpans (r:1 w:0)
-	// Storage: Staking Bonded (r:1 w:0)
-	// Storage: Staking Ledger (r:1 w:0)
-	// Storage: Staking Nominators (r:1 w:0)
-	// Storage: Staking ValidatorCount (r:1 w:0)
-	// Storage: ElectionProviderMultiPhase Round (r:1 w:0)
-	// Storage: ElectionProviderMultiPhase SnapshotMetadata (r:0 w:1)
-	// Storage: ElectionProviderMultiPhase DesiredTargets (r:0 w:1)
-	// Storage: ElectionProviderMultiPhase Snapshot (r:0 w:1)
-	// Storage: ElectionProviderMultiPhase CurrentPhase (r:0 w:1)
-	fn on_initialize_open_unsigned_with_snapshot() -> Weight {
-		(96_899_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(10 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
+		(30_788_000 as Weight)
+			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
+			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
```

Comment:
its not clear to me where all these reads and writes went to

Reply:
this benchmark used to be: create snapshot, and open signed phase. Now it is only "do the things for opening the signed phase". Again, this is reduced complexity showing up.